external/runtime/runtime.depproj

```diff
@@ -34,6 +34,9 @@
       <!-- Following is version used in release/uwp6.0 -->
       <Version Condition="'$(TargetGroup)'=='uap10.0.16299'">2.1.0-b-uwp6-25707-02</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.ILAsm">
```

Comment:
I see, in that case I think we should probably remove it from buildtools once this is merged then. Do you mind adding to this same PR one more PackageReference to `Microsoft.NETCore.ILDasm`? I added that to the buildtools depproj so we will need it here once this is merged, in fact, it is likely build will fail if we don't add that package reference.

Reply:
We can't remove it from `buildtools` until after all of CoreCLR is ported to use the msbuild-sdk as well.

I will update the PR so that `ILDasm` is also referenced, however 😄